targets/corefx.props

```diff
@@ -14,11 +14,13 @@
   <ItemGroup>
     <RepositoryReference Include="coreclr" />
     <RepositoryReference Include="standard" />
+    <EnvironmentVariables Include="DotNetSourceBuildRestoreSources=$(SourceBuiltPackagesPath)" Condition="'$(OfflineBuild)' != 'true'" />
```

Comment:
To make sure my plan is known: I'm planning on removing this and using the [`DotNetRestoreSourcePropsPath`](https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/auto-dependency-flow/api.md#-pdotnetrestoresourcepropspathpath) Repo API when I have my changes through.

Re: https://gitter.im/dotnet/source-build?at=59d29381614889d47567f95b
@eerhardt @weshaggard

Reply:
Yep.